adam/curriculum/imprecise_descriptions_curriculum.py

```diff
@@ -305,6 +314,139 @@ def make_jump_imprecise_temporal_descriptions(
     )
 
 
+def make_take_grab_imprecise_temporal_descriptions(
```

Comment:
Add unit test for the curriculum. See: https://github.com/isi-vista/adam/blob/master/tests/curriculum/imprecise_descriptions_curriculum_test.py

Also this should perhaps be renamed `make_take_grab_subtle_verb_descriptions` and added to a `make_subtle_verb_curriculum` function at the end of this file that mirrors the temporal return.

I've marked other instances of this with: 🧪